### DISCUSSION TITLE
issues open for solving

### DISCUSSION THREAD

**Discussion by GauravKumar9920** [JdeRobot/RoboticsAcademy#1930] [General]
Are there any issues related to ROS or the ones which haven't been assigned or solved yet.

**Comment by pariaspe** [comment]
Hi @GauravKumar9920 , thanks for your interest. What about #1914 and #1915?

**Reply by GauravKumar9920** [reply]
Okkayy I will look onto them

**Comment by pariaspe** [comment]
Hey, it will be also very useful to list the current available F1 models we have. We usually store them at [JdeRobot/CustomRobots](https://github.com/JdeRobot/CustomRobots) repo. After analyzing them, some improvement like JdeRobot/CustomRobots#192 can be made.

**Reply by GauravKumar9920** [reply]
sure, I will have a look at them

**Comment by GauravKumar9920** [comment]
Apart from custom robot (I am on it), are there any issues available @pariaspe

**Reply by pariaspe** [reply]
We renewed AutoParking exercise two weeks ago, could you try it and test that everything is working properly? It is not included in the latest RADI yet, you will have to build it manually

**Reply by pariaspe** [reply]
We can discuss it on issue #1944 that I've just created

**Reply by GauravKumar9920** [reply]
okkay I will try setting up an environment locally first and start testing it
